crates/astria-composer/src/collectors/geth.rs

```diff
@@ -206,24 +214,45 @@ impl Geth {
                             fee_asset_id: default_native_asset_id(),
                         };
 
+                        metrics::counter!(
+                            crate::metrics_init::TRANSACTIONS_RECEIVED,
+                            &[
+                                (ROLLUP_ID_LABEL, chain_name.clone()),
+                                (COLLECTOR_TYPE_LABEL, CollectorType::Geth.to_string())
```

Comment:
There is no advantage using a `CollectorType` enum over a hard coded `const COLLECTOR_TYPE: &str = "geth";`

Just replace this with a (module-private) const that's also `'static` so that you don't have to allocate here.